2024-11-26 11:45:18,345 - exec_batchAPI - INFO - ジョブステータス取得: failed
2024-11-26 11:45:18,345 - exec_batchAPI - ERROR - ジョブエラーの詳細: Errors(data=[BatchError(code='token_limit_exceeded', line=None, message='Enqueued token limit reached for gpt-4o-2024-11-20 in organization org-I1RzwRyPZjqKajv71LJjcHPF. Limit: 90,000 enqueued tokens. Please try again once some in_progress batches have been completed.', param=None)], object='list')
2024-11-26 11:45:18,345 - exec_batchAPI - ERROR - Batchオブジェクト: Batch(id='batch_67450a0c7fc881908df6142afe1d2084', completion_window='24h', created_at=1732577804, endpoint='/v1/chat/completions', input_file_id='file-NcCn5wdG1NX6oMnzQxMHPD', object='batch', status='failed', cancelled_at=None, cancelling_at=None, completed_at=None, error_file_id=None, errors=Errors(data=[BatchError(code='token_limit_exceeded', line=None, message='Enqueued token limit reached for gpt-4o-2024-11-20 in organization org-I1RzwRyPZjqKajv71LJjcHPF. Limit: 90,000 enqueued tokens. Please try again once some in_progress batches have been completed.', param=None)], object='list'), expired_at=None, expires_at=1732664204, failed_at=1732577805, finalizing_at=None, in_progress_at=None, metadata=None, output_file_id=None, request_counts=BatchRequestCounts(completed=0, failed=0, total=0))
